### ISSUE TITLE
[feature request]accept any string as compile options

### ISSUE THREAD

**Issue by xflywind** [PMunch/nim-playground-frontend#22]
accept any string from as compile options like goldbot does

https://godbolt.org

![image](https://user-images.githubusercontent.com/43030857/109615182-91f05d80-7b6e-11eb-86c3-d4596094c618.png)

**Comment by timotheecour** [comment]
I agree, the simplest would be showing a string like `nim c -r $file` which users could edit as they want, instead of having a complex UI with drop down boxes etc that's guaranteed to not stay in sync or be incomplete. This would allow users to pass arbitrary flags, eg `nim js -r $file`, `nim r --threads -d:foobar $file arg1 arg2` etc

I'm assuming it’s properly sandboxed, which it should be because users can just call shell from within nim program anyways.
